README.adoc

```diff
@@ -95,14 +95,31 @@ Most of the steps in this plugin require a common step variable called `remote`,
 |String
 |Encoding of input and output on the command or shell execution. Defaults to `UTF-8`.
 
-
 |proxy
 |Proxy, refer below.
 |If this is set, the proxy server is used to reach the remote host. Defaults to no proxy.
 
 |gateway
 |Remote
 |Gateway remote host. If this is set, the port-forwarding tunnel is used to reach the remote host. Defaults to no gateway.
+
+|appendName
+|boolean
+|If this is `true`, `name` is prefixed to each line in the log output. New format: `name\|log`.
+
+|logLevel
+|String
+a|Defaults to *SEVERE*
```

Comment:
Is the `a` supposed to be at the start of this line?

Reply:
That is actually asciidoc syntax to support asciidoc content in column (list in this case)

https://asciidoctor.org/docs/user-manual/#cols-format

Look for the string below to find the table
> The column styles are described in the table below.

Reply:
nice. I wondered if it might be related to the list.